vcd/resource_vcd_nsxt_network_dhcp_test.go

```diff
@@ -130,5 +138,7 @@ resource "vcd_nsxt_network_dhcp" "pools" {
     start_address = "7.1.1.130"
     end_address   = "7.1.1.140"
   }
+
+  dns_servers = ["8.8.4.4", "8.8.8.8"]
```

Comment:
Actually, we could use these in the examples (more private DNS):

```suggestion
  dns_servers = ["1.1.1.1", "1.0.0.1"]
```

Reply:
Ok, I'll change that!

Reply:
@lvirbalas already changed!